composer.json

```diff
@@ -1,54 +1,51 @@
 {
     "name":         "kokspflanze/bjy-authorize",
-    "description":  "Laminas\\Acl based firewall system for Laminas/ZF2/3 dispatch protection",
+    "description":  "Laminas\\Acl based firewall system for Laminas dispatch protection",
     "type":         "library",
     "license":      "BSD-3-Clause",
     "homepage":     "https://github.com/kokspflanze/BjyAuthorize",
     "keywords": [
         "laminas",
-        "zf",
-        "zf2",
-        "zf3",
         "acl",
-        "lmc-user",
-        "zfc-user"
+        "authorization",
+        "lmc-user"
     ],
     "authors": [
         {
-            "name":      "Ben Youngblood",
-            "email":     "bx.youngblood@gmail.com",
-            "homepage":  "http://bjyoungblood.com/",
-            "role":      "Developer"
+            "name":     "Ben Youngblood",
+            "email":    "bx.youngblood@gmail.com",
+            "homepage": "http://bjyoungblood.com/",
+            "role":     "Developer"
         },
         {
-            "name":      "Marco Pivetta",
-            "email":     "ocramius@gmail.com",
-            "homepage":  "http://ocramius.github.com/",
-            "role":      "Developer"
+            "name":     "Marco Pivetta",
+            "email":    "ocramius@gmail.com",
+            "homepage": "http://ocramius.github.com/",
+            "role":     "Developer"
         }
     ],
     "require": {
-        "php": "^7.3",
-        "laminas/laminas-permissions-acl": "^2.7",
-        "laminas/laminas-mvc":             "^3.2",
-        "laminas/laminas-eventmanager":    "^3.3",
-        "laminas/laminas-servicemanager":  "^3.6",
-        "laminas/laminas-http":            "^2.14",
-        "laminas/laminas-view":            "^2.12",
-        "laminas/laminas-authentication":  "^2.7",
-        "laminas/laminas-cache":           "^2.10"
+        "php": "^7.3 || ~8.0.0",
```

Comment:
why not `^7.3 || ^8.0`

Reply:
@kokspflanze Because `^7.3 || ^8.0` would immediatly release this package for PHP 8.1 as well, but we can't currently run the unit tests with 8.1. I tried it, but many dependencies are not 8.1-ready yet, see https://github.com/driehle/BjyAuthorize/runs/3700749417?check_suite_focus=true.

Therefore, `~8.0.0` ensures that only 8.0 is allowed. The semantic of `~` is explained [here](https://getcomposer.org/doc/articles/versions.md#tilde-version-range-). Technically, we could instead write `8.0.*`, but `~8.0.0` is more explicit as it specifies the minimum version. Seems to be like a best practice convetion, if you look at the laminas packages you will see that pretty much all of them use this syntax, e.g. [laminas-permission-acl](https://packagist.org/packages/laminas/laminas-permissions-acl).